tests/components/zwave_js/test_sensor.py

```diff
@@ -60,26 +54,26 @@ async def test_numeric_sensor(hass, multisensor_6, integration):
     assert state
     assert state.state == "9.0"
     assert state.attributes["unit_of_measurement"] == TEMP_CELSIUS
-    assert state.attributes["device_class"] == DEVICE_CLASS_TEMPERATURE
+    assert state.attributes["device_class"] == SensorDeviceClass.TEMPERATURE
```

Comment:
Maybe also import and use `ATTR_UNIT_OF_MEASUREMENT`?

Reply:
Agreed, done.